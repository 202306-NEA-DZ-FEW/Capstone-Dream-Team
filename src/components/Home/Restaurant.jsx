export default function Restaurant() {
    return (
        <div class=' mt-20 bg-gradient-to-bl from-[#FCDBB4] to-violet-50 flex items-center justify-center '>
            <div class='container mx-auto p-4'>
                <h2 className='max-w-lg mb-6 font-sans text-2xl text-center font-bold leading-none tracking-tight text-[#192655] sm:text-3xl md:mx-auto'>
                    Join Us - Restaurants can make a Difference
                </h2>
                <p className='text-base text-center text-gray-700 md:text-lg mb-6'>
                    As a restaurant owner or manager, you play a vital role in
                    our mission. We are reaching out to you with an exciting
                    opportunity to not only make a positive impact on your
                    community but also promote your restaurant to a wider
                    audience. Here is how you can participate:
                </p>
                <div class='grid grid-cols-1 sm:grid-cols-2 md:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4 gap-4'>
                    <div class='bg-white rounded-lg border p-4'>
                        <img
                            src='images/home/restaurant/pics of food.jpg'
                            alt='Placeholder Image'
                            class='w-full h-48 rounded-md object-cover'
                        ></img>
                        <div class='px-1 py-4'>
                            <div class='font-bold text-xl mb-2'>
                                List Your Meals
                            </div>
                            <p class='text-gray-700 text-base'>
                                By joining our cause, you can easily showcase
                                your restaurant and its delectable dishes on our
                                website. We will create a dedicated page for
                                your restaurant, complete with mouthwatering
                                images and descriptions of your meals.
                            </p>
                        </div>
                    </div>
                    <div class='bg-white rounded-lg border p-4'>
                        <img
                            src='images/home/restaurant/accept.jpg'
                            alt='Placeholder Image'
                            class='w-full h-48 rounded-md object-cover'
                        ></img>
                        <div class='px-1 py-4'>
                            <div class='font-bold text-xl mb-2'>
                                Accept Coupons
                            </div>
                            <p class='text-gray-700 text-base'>
                                Donors who want to support our mission can
                                purchase meal coupons from your restaurant.
                                These coupons can be redeemed by those in need,
                                ensuring that you reach a broader customer base.
                            </p>
                        </div>
                    </div>
                    <div class='bg-white rounded-lg border p-4'>
                        <img
                            src='images/home/restaurant/controle.jpg'
                            alt='Placeholder Image'
                            class='w-full h-48 rounded-md object-cover'
                        ></img>
                        <div class='px-1 py-4'>
                            <div class='font-bold text-xl mb-2'>
                                Dashboard Control
                            </div>
                            <p class='text-gray-700 text-base'>
                                We provide you with a secure and user-friendly
                                dashboard that allows you to verify and track
                                coupon redemptions. You will have full control
                                over the process and can monitor the impact your
                                contributions are making.{" "}
                            </p>
                        </div>
                    </div>
                    <div class='bg-white rounded-lg border p-4'>
                        <img
                            src='images/home/restaurant/delete.jpg'
                            alt='Placeholder Image'
                            class='w-full h-48 rounded-md object-cover'
                        ></img>
                        <div class='px-1 py-4'>
                            <div class='font-bold text-xl mb-2'>
                                Coupon Deletion
                            </div>
                            <p class='text-gray-700 text-base'>
                                Once a coupon is used and a meal is delivered to
                                someone in need, you can easily mark it as
                                redeemed in your dashboard. This ensures that
                                the coupons are used only once, maintaining the
                                integrity of the program.{" "}
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    );
}
