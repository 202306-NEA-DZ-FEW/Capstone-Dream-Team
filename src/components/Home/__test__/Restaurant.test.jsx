import renderer from "react-test-renderer";

import Restaurant from "../Restaurant";

it("rend correctement", () => {
    const tree = renderer.create(<Restaurant />).toJSON();
    expect(tree).toMatchSnapshot();
});
