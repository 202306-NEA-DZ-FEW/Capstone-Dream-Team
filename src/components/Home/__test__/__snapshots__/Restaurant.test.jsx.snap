// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rend correctement 1`] = `
<div
  className="pt-20 bg-gradient-to-bl bg-white flex items-center max-w-screen-xl justify-center scroll-smooth "
>
  <div
    className="container mx-auto p-4"
  >
    <h2
      className="hover:-translate-y-3 duration-300 max-w-lg mb-10 font-sans text-2xl text-center font-bold leading-none tracking-tight text-[#192655] sm:text-3xl lg:text-4xl mx-auto"
    >
      home.restaurant.title
    </h2>
    <p
      className="text-base text-center text-gray-700 md:text-lg mb-16"
    >
      home.restaurant.subtitle
    </p>
    <div
      className="grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4 gap-8"
    >
      <div
        className="hover:-translate-y-3 duration-300 cursor-pointer hover:shadow-2xl bg-white rounded-lg border p-4"
      >
        <img
          alt="take pictures of meal"
          class="w-full just h-48 rounded-md object-cover"
          src="images/home/restaurant/res1-removebg-preview.png"
        />
        <div
          class="py-4 text-center"
        >
          <div
            class="font-bold text-orange-600 text-xl mb-2"
          >
            home.restaurant.step1
          </div>
          <p
            class="text-gray-700 text-base"
          >
            home.restaurant.step1text
          </p>
        </div>
      </div>
      <div
        class="hover:-translate-y-3 duration-300 cursor-pointer hover:shadow-2xl bg-white rounded-lg border p-4"
      >
        <img
          alt="accept coupon"
          class="w-full h-48 rounded-md object-cover"
          src="images/home/restaurant/res2-removebg-preview.png"
        />
        <div
          class="py-4 text-center"
        >
          <div
            class="font-bold text-orange-600 text-xl mb-2"
          >
            home.restaurant.step2
          </div>
          <p
            class="text-gray-700 text-center text-base"
          >
            home.restaurant.step2text
          </p>
        </div>
      </div>
      <div
        class="hover:-translate-y-3 duration-300 cursor-pointer hover:shadow-2xl bg-white rounded-lg border p-4"
      >
        <img
          alt="dashboard control"
          class="w-full h-48 rounded-md object-cover"
          src="images/home/restaurant/res3-removebg-preview.png"
        />
        <div
          class="py-4 text-center"
        >
          <div
            class="font-bold text-orange-600 text-xl mb-2"
          >
            home.restaurant.step3
          </div>
          <p
            class="text-gray-700 text-base"
          >
            home.restaurant.step3text
          </p>
        </div>
      </div>
      <div
        className="hover:-translate-y-3 duration-300 cursor-pointer hover:shadow-2xl bg-white rounded-lg border p-4"
      >
        <img
          alt="delete"
          class="w-full h-48 rounded-md object-cover"
          src="images/home/restaurant/res4-removebg-preview.png"
        />
        <div
          class="py-4 text-center"
        >
          <div
            class="font-bold text-orange-600 text-xl mb-2"
          >
            home.restaurant.step4
          </div>
          <p
            class="text-gray-700 text-base"
          >
            home.restaurant.step4text
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
